.docker/base/Dockerfile

```diff
@@ -4,33 +4,47 @@ LABEL maintainer "Irina Spiridonova <irinasp@microsoft.com>; Darren Gehring <dar
 USER everest
 WORKDIR ${MYHOME}
 
-#PAT token should be provided in the docker run command or as a variable in VSTS build definition 
-ENV PAT $PAT
+ENV CI_LOGS ${MYHOME}"/ci-logs/"
+
+#PAT token and agent name should be provided in the docker run command when docker container is started 
```

Comment:
can you add a comment saying which variables exactly the user should provide when running the `docker create` command?